index.html

```diff
@@ -10,16 +10,170 @@
+                    </div>
+                    <div>
+                        <div class="primary-color title">
+                            Lecture
+                        </div>
+                        <div class="content">Listen to the sorcerers and prophets about the various spells and
+                            incantations used by demigods themselves
+                        </div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="flex justify-center">
+                        <img class="icon" src="assets/lecture.svg" alt="practical">
+                    </div>
+                    <div>
+                        <div class="primary-color title">
+                            Practical
+                        </div>
+                        <div class="content">The chance to test and practice various kind of spells including
+                            incantations and prepare for battle!</div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="flex justify-center">
+                        <img class="icon" src="assets/lecture.svg" alt="">
+                    </div>
+                    <div>
+                        <div class="primary-color title">
+                            Radhn Festival
+                        </div>
+                        <div class="content">General Radhn, the mightiest demigod of the shattering awaits you.
+                            Champions, prepare for battle. A celebration of war! THE RADHN FESTIVAL!!!</div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="flex justify-center">
+                        <img class="icon" src="assets/lecture.svg" alt="">
+                    </div>
+                    <div>
+                        <div class="primary-color title">
+                            Reward
+                        </div>
+                        <div class="content">Defeat the General. Claim glory and grab the great rune. The reward is THE
+                            STARSCOURAGE GREATSWORD!</div>
+                    </div>
+                </div>
+            </div>
+            <div class="underline">SEE THE WHOLE PROGRAM</div>
+        </div>
+    </section>
+    <section id="teachers" class="flex justify-center teachers">
+        <div class="bg">
+            <header class="flex justify-center">
+                <div>
+                    <div>
+                        Featured Teachers
+                    </div>
+                    <div class="flex justify-center">
+                        <div class="divider"></div>
+                    </div>
+                </div>
+            </header>
+            <div id="speakers-container" class="flex justify-center wrap">
+                <!-- js objects go here -->
+            </div>
```

Comment:
- [ ] Please add the image behind the featured teacher's images as it's on the given template and make sure it matches the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/174877433-6da5e9bf-91bd-49cb-b65c-0f5f38ab0063.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/174877467-a5b9cbeb-44ae-4619-a3f2-28e69c4bfaf1.png)

Reply:
I've checked the element on that site. You can check yourself here. It's not web design. It's the image itself. https://summit.cckorea.org/main.html

It's just a single image element without stacking with underground style. Just a simple single image. That design is with the image.

Reply:
@HtetNaing0814 Kindly implement the changes requested, the styling is part of the requirement to achieve a design similar to the image given.